tensorrt_llm/serve/router.py

```
@@ -562,17 +591,32 @@ async def _monitor_servers(self, poll_interval: float = 10.0):
                         logger.debug(
                             f"No change in {self._server_role} server list: {len(self._servers)} servers"
                         )
+                self._last_successful_poll = time.monotonic()
+            except asyncio.CancelledError:
+                raise
             except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)